test/suites/snapshots.sh

```diff
@@ -7,35 +7,59 @@ test_snapshots() {
   lxc init testimage foo
 
   lxc snapshot foo
-  [ -d "${LXD_DIR}/snapshots/foo/snap0" ]
```

Comment:
No, it doesn't with zfs. zfs doesn't make this directory until you actually start the snapshot (viz. my fix above)